ajax/libs/hola-video.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "hola-video.js",
+  "description": "An HTML5 and Flash video player with a common API and skin for both. This is modified version to support HolaCDN.",
+  "version": "5.10.2-21",
+  "license": "Apache-2.0",
+  "keywords": [
+    "flash",
+    "html5",
+    "player",
+    "video",
+    "videojs"
+  ],
+  "homepage": "https://holacdn.com/player",
+  "author": "Steve Heffernan",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hola/video.js.git"
+  },
+  "npmName": "@hola.org/video.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
@sreym Because we don't want to fetch `examples` directory, let's modify it to:
```json
"files": [
  "!(examples)/*",
  "!(examples)"
]
```
Thanks.